templates/internet-of-things/core.html

```diff
@@ -0,0 +1,1171 @@
+{% extends "templates/base.html" %}
+
+{% block title %} Ubuntu Core {{ page_title }}{% endblock %}
+{% block meta_description %}Ubuntu Core，Ubuntu物联网，Ubuntu嵌入式，Core20开发{{ meta_description }}{% endblock %}
+
+{% block content %}
+{% include "internet-of-things/secondary-nav.html" %}
+
+  <section class="p-strip-suru-half-top">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>安全的嵌入式设备</h1>
+        <h2 class="p-heading--four u-sv3">为您的设备配备自己的应用商店</h2>
+        <p>
+          <a class="p-button--positive" href="https://ubuntu.com/core/smartstart">快速进入市场</a>
+        </p>
+        <p class="u-sv3">
+          <a class="p-link--external p-link--inverted" href="https://assets.ubuntu.com/v1/b2c770ea-Ubuntu+Core20+Datasheet.pdf">阅读Ubuntu Core手册</a>
+        </p>
+      </div>
+      <div class="col-5 u-align--center u-embedded-media">
+        <iframe title="UC20 video" class="u-embedded-media__element" src="//player.bilibili.com/player.html?aid=501588719&bvid=BV1YN411R7pY&cid=295296600&page=1" frameborder="0" framespacing="0" allowfullscreen=""></iframe>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light u-no-padding--top">
+    <div class="row u-vertically-center u-align--center">
```

Comment:
The `u-align--center` here is stoping the row doing its thing. Can you add a `col-12` div inside with the align class